### PR TITLE
Feature/count of teams

### PR DESCRIPTION
**Pull request by austincarrjones**
I was unable to get the class variable `@@teams` to work, so I may have to rework that later. (It kept storing values from create_teams, even from separate tests (it blocks).

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,9 +1,4 @@
-SimpleCov.start
-
-RSpec.configure do |config|
-    config.formatter = :documentation
-end
-
+require 'simplecov'
```
Comment (noahdurbin) [inline spec/spec_helper.rb]: sorry we told you we didn't need this!

File: lib/team_factory.rb [hunk 2]
```diff
@@ -16,4 +16,9 @@ def create_teams
     end
     @teams
   end
+  
+  def count_of_teams
```
Comment (noahdurbin) [inline lib/team_factory.rb:20]: I was including methods in the non factory class but this definitely works, I was actually wondering if we might want to abstract the methods out into a separate class altogether.

**Review by noahdurbin — APPROVED** [review]
looks good!

File: lib/team_factory.rb (hunk 2 above)
Comment (zach-bergman) [inline lib/team_factory.rb:20]: Yes we can talk about abstracting methods into their own respective classes! But this works in the TeamFactory class for now! Very nice

**Review by zach-bergman — APPROVED** [review]
Nice work! Just handled the merge conflicts with the Gemfile and spec_helper, hopefully I didn't mess anything up :)

**Pull request merged**